packages/fluentui/docs/src/examples/components/Pill/Types/PillsExample.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+import { Pill, Pills } from '@fluentui/react-northstar';
+
+const PillsExample = () => (
+  <Pills>
```

Comment:
just small improvement, please if you could add some aria-label for the "listbox".
And I think we should mention it in do "Do" section that for best screen reader experience would be good to label <Pills>(listbox) container.

Reply:
Done